pkg/reconciler/gc/v2/gc.go

```diff
@@ -113,7 +116,7 @@ func nonactiveRevisions(revs []*v1.Revision, config *v1.Configuration) []*v1.Rev
 	for i := 0; i < swap; {
 		if isRevisionActive(revs[i], config) {
 			swap--
-			revs[swap], revs[i] = revs[i], revs[swap]
+			revs[i] = revs[swap]
```

Comment:
the comment feels outdated with this new semantic.

Reply:
done.